x/adminmodule/types/keys.go

```diff
@@ -48,7 +53,17 @@ func ProposalKey(proposalID uint64) []byte {
 	return append(ProposalsKeyPrefix, GetProposalIDBytes(proposalID)...)
 }
 
+// ProposalLegacyKey gets a specific proposal from the store
+func ProposalLegacyKey(proposalID uint64) []byte {
+	return append(ProposalsKeyLegacyPrefix, GetProposalIDBytes(proposalID)...)
+}
+
 // ActiveProposalQueueKey returns the key for a proposalID in the activeProposalQueue
 func ActiveProposalQueueKey(proposalID uint64) []byte {
 	return append(ActiveProposalQueuePrefix, GetProposalIDBytes(proposalID)...)
 }
+
+// ActiveProposalLegacyQueueKey returns the key for a proposalID in the activeProposalLegacyQueue
+func ActiveProposalLegacyQueueKey(proposalID uint64) []byte {
```

Comment:
both active proposals related funcs (as well as respective keys) are not needed anymore